src/components/flag/Flag.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Flag } from './Flag';
+
+describe('Test Flag component', () => {
+  it('should render Flag component', () => {
+    render(
+      <Flag name="Ukraine" code="UA" />
+    );
+    expect(screen.getByAltText('Ukraine')).toBeInTheDocument();
+  });
+
+  it('should render Flag component by class flag-img & by props isMobile & props isGreetingCountry', () => {
+    const { container } = render(
+      <Flag name="Ukraine" code="UA" isMobile isGreetingCountry />
+    );
+    expect(container.getElementsByClassName('flag-img')[0]).toBeInTheDocument();
+    expect(container.getElementsByClassName('flag-img_mobile')[0]).toBeInTheDocument();

```

Comment:
Add another test to check the case when flag-img_desktop class is applied and without flag-img_greeting-country class